src/PaymentGateways/PayPalCommerce/AdminSettingFields.php

```diff
@@ -69,7 +79,9 @@ public function payPalCommerceAccountManagerField() {
 							);
 							?>
 						</button>
-						<a class="give-hidden" target="_blank" data-paypal-onboard-complete="givePayPalOnBoardedCallback" href="#" data-paypal-button="true">
+						<a class="give-hidden" target="_blank"
```

Comment:
@JasonTheAdams are these changes because of auto formatting?

Reply:
Yeah, my IDE wraps lines at 120 characters.

Reply:
Per our conversation on Slack, I'm going to keep the wrapping: https://givewp.slack.com/archives/CM9NDF2QY/p1596818230008900?thread_ts=1596816407.002500&cid=CM9NDF2QY